python/cudf/cudf/core/column/string.py

```diff
@@ -73,12 +73,17 @@ def wrapper(*args, **kwargs):
                         )
                     return ret
 
+                wrapper.__doc__ = passed_attr.__doc__
```

Comment:
Suggest using `functools.wraps` instead of this, e.g., (https://github.com/rapidsai/cudf/blob/branch-0.10/python/cudf/cudf/utils/utils.py#L189).

`wraps` does some other nice things in addition to copying `__doc__`:

https://docs.python.org/2/library/functools.html#functools.update_wrapper